Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1747,6 +1747,14 @@ extension BrowserViewController: SearchViewControllerDelegate {
 
         self.present(navController, animated: true, completion: nil)
     }
+
+    func searchViewController(_ searchViewController: SearchViewController, didHighlightText text: String, search: Bool) {
+        self.urlBar.setLocation(text, search: search)
+    }
+
+    func searchViewControllerDidFinishHighlighting(_ searchViewController: SearchViewController) {
+        searchViewController.resignFirstResponder()
```

Comment:
This is called when the user has typed DOWN, then UP — i.e. focus should move back to the location bar.

Also tried: explicitly setting BVC or the location bar as first responder.